<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>P  L  A  N  E  T</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QDockWidget" name="settingsDock">
   <property name="minimumSize">
    <size>
     <width>275</width>
     <height>650</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>P L A N E T   O P T I O N S</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QGroupBox" name="terrainDock">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>231</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>T E R R A I N</string>
     </property>
     <widget class="QSlider" name="frequency">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>32</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="persistance">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="amplitude">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="feqText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>Frequency</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="freqDisplay">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="perText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Persistance</string>
      </property>
     </widget>
     <widget class="QLabel" name="ampText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Amplitude</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="persistanceDisplay">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="amplitudeDisplay">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="octText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Octaves</string>
      </property>
     </widget>
     <widget class="QSlider" name="octaves">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="octavesDisplay">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>200</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QGroupBox" name="waterDock">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>W A T E R</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="levelsText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>Levels</string>
      </property>
     </widget>
     <widget class="QSlider" name="waterLevels">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="waterDisplay">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QGroupBox" name="moonDock">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>M O O N S</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QSlider" name="moonNumber">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="moonDisplay">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="generate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>490</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate Planet</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="normalMap">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>570</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Normal </string>
     </property>
    </widget>
    <widget class="QCheckBox" name="textureMap">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>570</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Texture</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>530</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="wireframe">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>600</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Wireframe </string>
     </property>
    </widget>
    <widget class="QCheckBox" name="freezeTime">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>600</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Freeze time</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
